### PR TITLE
[Onprem] Sky Queue Logging

### PR DESCRIPTION
**Pull request by michaelzhiluo**
Fixes bug discovered by @concretevitamin.
## Before:
`sky queue on-prem-test`
```
Fetching and parsing job queue...

Job queue of cluster on-prem-test
ID  NAME     SUBMITTED    STARTED      DURATION  RESOURCES     STATUS     LOG                                        
1   sky-cmd  43 mins ago  43 mins ago  1s        1x [CPU:0.5]  SUCCEEDED  ~/sky_logs/sky-2022-08-02-22-31-40-033756  

Local cluster azure-onprem-test1 is uninitialized; skipped.
Local cluster on-prem-smoke-test is uninitialized; skipped.
Local cluster dev-production is uninitialized; skipped.
Local cluster azure-onprem is uninitialized; skipped.
```

## After:
`sky queue on-prem-test`
```
Fetching and parsing job queue...

Job queue of cluster on-prem-test
ID  NAME     SUBMITTED    STARTED      DURATION  RESOURCES     STATUS     LOG                                        
1   sky-cmd  43 mins ago  43 mins ago  1s        1x [CPU:0.5]  SUCCEEDED  ~/sky_logs/sky-2022-08-02-22-31-40-033756  
```

### CODE REVIEW

File: sky/cli.py [hunk 1]
```diff
@@ -1120,7 +1120,7 @@ def queue(clusters: Tuple[str], skip_finished: bool, all_users: bool):
         _show_job_queue_on_cluster(cluster, handle, backend, code)
 
     for local_cluster in local_clusters:
-        if local_cluster not in clusters:
+        if local_cluster not in clusters and len(clusters) > 1:
```
Comment (concretevitamin) [inline sky/cli.py:1123]: Mind explaining this change?  What happens if I have say clusters A, B, C which are all local + uninitialized, and I do `sky queue A B` (or `sky queue my-cloud-clus A`)? Would I see the message "... C is uninitialized?"

Comment (michaelzhiluo) [reply sky/cli.py]: Fixed it, `C is uninitialized` should not show when multiple clusters are specified. These messages will only appear when `sky queue` is specified without specific clusters.

**Review by concretevitamin — APPROVED** [review]
Thanks @michaelzhiluo.  I think this is okay to go in, but I find it pretty weird that we have inconsistent logging:
```
» sky queue on-prem-smoke-test
Fetching and parsing job queue...
Found uninitialized local cluster on-prem-smoke-test. Run this command to initialize it locally: sky launch -c on-prem-smoke-test ''

» sky queue
Fetching and parsing job queue...
...
Local cluster on-prem-smoke-test is uninitialized; skipped.
```
Do you see a clean way to make them consistent?

**Comment by michaelzhiluo** [comment]
@concretevitamin Feel free to push back, I think it is reasonable logging, since these are two different cases, the first trying to go through all clusters (and hence the abbreviated error message) and the second trying to directly query an un-initialized cluster. Otherwise, I can copy and paste the same error message for both cases.

**Pull request merged**